eisbuk-admin/package.json

```diff
@@ -87,7 +89,7 @@
     "@types/jest": "^26.0.23",
     "@types/lodash": "^4.14.170",
     "@types/luxon": "^2.0.5",
-    "@types/node": "^12.0.0",
+    "@types/node": "^16.11.10",
```

Comment:
We should upgrade everything to node 16. Currently our `.nvmrc` states `v14.18.1` and `functions/package.json` specifies `14`. Node 16 in Firebase functions is supported as a beta release; we should upgrade to that version.

Not so much a comment about this work, just mentioning.